src/promptflow/promptflow/_sdk/entities/_eager_flow.py

```diff
@@ -28,6 +28,12 @@ def __init__(
         code = Path(code)
         # entry function name
         self.entry = entry
+        # entry file name
+        try:
+            # TODO: remove this after gen meta removed entry file
```

Comment:
what do we mean by this remove?

Reply:
I plan to remove the entry generation logic after executor can generate meta according to the entry. As the latest discussion, we still need this logic to check if the entry is a module or a local file.